src/java/org/apache/cassandra/db/SortedLocalRanges.java

```diff
@@ -91,7 +91,7 @@ static SortedLocalRanges create(ColumnFamilyStore cfs)
 
         do
         {
-            tmd = storageService.getTokenMetadata();
+            tmd = cfs.keyspace.getReplicationStrategy().getTokenMetadata();
```

Comment:
If token metadata is the reason for changing `CompactionRealm` to `CFS`, how about adding `getTokenMetadata()` to `CompactionRealm`?

Reply:
Good idea.

Reply:
Actually I don't need to add it and still use Realm. I am fixing it.